documentation/SupportingDocuments/API_Documentation/User Stories/Edge Cloud APIs UserStory.md

```diff
@@ -0,0 +1,181 @@
+|Post-conditions|The UE is connected to the most adequate ESP, it may request to reconnect (repeat Step 7) at any time if the perceived quality is not adequate| M |
+
+
+
+
+## Flow
+```mermaid
+sequenceDiagram
+    participant UC
+    participant AP
+    participant Operator
+    participant ESP(Hyperscaler)
+    Note over AP,Operator: PRE App on device attached <br/> to operator network
+    Note over AP,Operator: PRE App developer registered, <br/> authenticated and authorised
+    UC->>AP: The user client request access to the application
+         Note over Operator,Operator: SED API
+    AP->>Operator: GET/ AP Invokes Simple Edge Discovery API to get the closest Edge cloud zone for this MSISDN
+    Operator->>AP: Returns the name of the closest Edge cloud zone for this MSISDN(edgeCloudZoneInfo)
+      AP->>ESP(Hyperscaler): The AP instantiates the App in the Edge Cloud Zone determinated by the SED API
+      ESP(Hyperscaler)->>AP: Return EdgeCloudZoneName and EdcloudZoneProvider
+     AP->>UC: The AP provides the URL of the closest Edge Cloud Zone to the UC.
+  alt 
+     Note over Operator,Operator: TI API
+     AP->>ESP(Hyperscaler): AP invokes traffic influence API to activate the appropriate routing at the Edge of the mobile network toward the local ESP Instance for the Premium Users
+     ESP(Hyperscaler)->>AP: The traffic flow is routed from the selected UPF to the requested ESP Instance for the selected users
+   end
+```
+
+
+#### Scenario 2: Request triggered by the AP and the Operator will host the App.
+
+
+| Item | Description | Support Qualifier |
+|----|----|----|
+|Summary|Edge Service Providers offers an Edge  platform that exposes computing and networking services in a given territory with several levels of abstraction being the Edge Cloud Zone the lowest one. Edge Service Providers offers a service through EdgeCloud APIs that permits application providers to deploy instances of an application in one or several Edge Cloud Zones, and to Applications Users the posibility to chose the most appropriate Edge Cloud Zone available at a given moment| M |
+|Pre-conditions|The AP has a resource usage agreement with the owner of the Edge Platform| M |
+|Begins when|The Application Provider (AP) invokes the Edge Application Management API to submit an application with references to obtain all needed data for their instantiation (container images or VM images and manifests describing required resources by an application)| M |
+|Step 1|The Operator authorizes the request (terminates with a notification to the AP if it is not valid)| M |
+|Step 2|The Operator validates the request (terminates with a notification to the AP if content is not valid)| M|
+|Step 3|The Operator stores the app information and returns an Id for reference (terminates with a notification to the AP if error)| M |
+|Step 4|The Edge Application Management API instantiate the application in an specific Edge Cloud Zone or serverals | M |

```

Comment:
I think we should mention who (the actor) initiates the Edge Application Management API.